<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CProjectDialog</class>
 <widget class="QDialog" name="CProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_pALabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Project A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="m_pBLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Project B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="CProjectListWidget" name="m_pAListItem"/>
   </item>
   <item row="1" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" rowspan="4">
    <widget class="CProjectListWidget" name="m_pBListItem"/>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="m_pLeftBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Cplusplus_gui_programming_with_Qt4/qt-book-examples/qt-book/chap09/projectchooser/projectchooser.qrc">
       <normaloff>:/images/leftarrow.png</normaloff>:/images/leftarrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="m_pRightBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Cplusplus_gui_programming_with_Qt4/qt-book-examples/qt-book/chap09/projectchooser/projectchooser.qrc">
       <normaloff>:/images/rightarrow.png</normaloff>:/images/rightarrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProjectListWidget</class>
   <extends>QListWidget</extends>
   <header>projectlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Cplusplus_gui_programming_with_Qt4/qt-book-examples/qt-book/chap09/projectchooser/projectchooser.qrc"/>
 </resources>
 <connections/>
</ui>
